ibis/expr/types/relations.py

```diff
@@ -2957,6 +2958,415 @@ def pivot_longer(
 
         return self.select(~pivot_sel, **new_cols)
 
+    @util.experimental
+    def pivot_wider(
+        self,
+        *,
+        id_cols: s.Selector | None = None,
+        names_from: str | Iterable[str | s.Selector] = "name",
+        names: Iterable[str] | None = None,
+        names_prefix: str = "",
+        names_sep: str = "_",
+        names_sort: bool = False,
+        values_from: str | Iterable[str | s.Selector] = "value",
+        values_fill: int | float | str | ir.Scalar | None = None,
```

Comment:
maybe this is just `fill`?

Reply:
Simlilar to [prefixes for `names`-related things](https://github.com/ibis-project/ibis/pull/5768#discussion_r1142295077) I'd like to keep the `values_` prefix.